db/migrate/20190923170526_add_user_id_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToEvents < ActiveRecord::Migration[5.1]
+  def change
+    add_column :events, :user_id, :integer
```

Comment:
`[Optional]` It would have been better if you were called the `:user_id` as `:creator_id` to make a difference between the creator of the event and the user attending the event.

Reply:
Changed it as requested.